lib/resources/dstu2/condition.yaml

```diff
@@ -0,0 +1,133 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/condition-definitions.html#Condition
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: Condition
+  note: resourceType must be Condition
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "p123412"}'
+  note: The id value must match the Condition/&lt;id> value.
+  url: http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id
+  action: update
+
+- name: patient
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference (Patient)
+  description: Who has the condition.
+  example: |
+             {"patient": {
+                "reference":"Patient/2744010",
+                "display":"RHEUM, TEST ONE""}}
+
+- name: encounter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Encounter during which the condition was first asserted.
+  example: |
+             {"encounter":{
+                "reference":"Encounter/1309819"}}
+
+- name: asserter
+  required: 'No'
+  cardinality: 0..1
+  type: Reference (Practitioner)
+  description: Person who takes responsibility for asserting the existence of the condition as part of the electronic record.
+  note: Asserter must be a `Practitioner` reference.
+  example: |
+             {"asserter":{
+                "reference":"Practitioner/2770007",
+                "display":"Song, River"}}
+
+- name: dateRecorded
+  required: 'No'
+  cardinality: 0..1
+  type: date
+  description: Estimated or actual date the condition/problem/diagnosis was first entered.
+  example: '{"dateRecorded":"2013-02-08"}'
+
+- name: code
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: Identification of the condition, problem or diagnosis.
+  example: |
+             {"code":{
+                "coding":[{
+                  "system":"http://hl7.org/fhir/sid/icd-9",
+                  "code":"345.0",
+                  "display":"Generalized nonconvulsive epilepsy",
+                  "userSelected":true}],
+                "text":"Absence seizures"}}
```

Comment:
For consistency can you update your examples so that
- There is a single space following each colon
- The opening grouper { or [ is on the initial line and subsequent groupers are on the start of the following line. It makes it look cleaner when your example contains multiple hashes in a list.

```
{"code": {
    "coding": [
      {"system": "http: //hl7.org/fhir/sid/icd-9",
       "code": "345.0",
       "display": "Generalized nonconvulsive epilepsy",
       "userSelected": true},
      {"system": "http: //hl7.org/fhir/sid/icd-9",
       "code": "345.0",
       "display": "Generalized nonconvulsive epilepsy",
       "userSelected": true}],
    "text": "Absence seizures"}}
```